timemachine/md/minimizer.py

```diff
@@ -115,19 +113,23 @@ def shift(d, dr):
     return np.asarray(opt_state.position)
 
 
-def minimize_host_4d(
+def pre_equilibrate_host(
     mols: List[Chem.Mol],
     host_config: HostConfig,
     ff: Forcefield,
     mol_coords: Optional[List[NDArray]] = None,
-    windows: int = 50,
-    n_steps_per_window: int = 50,
-) -> np.ndarray:
-    """
-    Insert mols into a host system via 4D decoupling using Fire minimizer at lambda=1.0,
-    0 Kelvin Langevin integration at a sequence of lambda from 1.0 to 0.0, and Fire minimizer again at lambda=0.0
+    minimizer_steps_per_window: int = 500,
+    minimizer_windows: int = 2,
+    equilibration_steps: int = 1000,
```

Comment:
nit: seems possibly useful to expose `max_lambda` for the minimizer here too

Reply:
Exposed in https://github.com/proteneer/timemachine/pull/1339/commits/52d65dc1a9fcd7eb8dc6e9a2d4a8a6b6a14db2a0